services/QuillLMS/app/workers/snapshots/cache_snapshot_top_x_worker.rb

```diff
@@ -51,8 +51,8 @@ def perform(cache_key, query, user_id, timeframe, school_ids, filters)
     end
 
     private def generate_payload(query, timeframe, school_ids, filters)
-      timeframe_start = DateTime.parse(timeframe['current_start'])
```

Comment:
This change is from https://github.com/empirical-org/Empirical-Core/pull/11138 and does not need to be reviewed here.